R/setup.R

```diff
@@ -33,7 +33,7 @@ update_rextendr_version <- function(path, cur_version = NULL) {
 }
 
 update_sys_reqs <- function(path) {
-  cur <- "Cargo (rustc package manager)"
+  cur <- "Cargo (Rust's package manager), rustc"
```

Comment:
From [Using Rust in CRAN Packages](https://cran.r-project.org/web/packages/using_rust.html) 
> "SystemRequirements: Cargo (Rust's package manager), rustc"

This update is required. Thanks!